includes/Media.php

```diff
@@ -70,7 +70,7 @@ class Media {
 	 *
 	 * @var string
 	 */
-	const POSTER_POST_META_KEY = 'amp_is_poster';
+	const POSTER_POST_META_KEY = 'web_stories_is_poster';
```

Comment:
@swissspidy Removed the under score here, as that makes the meta key protected and not updatable via the REST API